noir-projects/aztec-nr/aztec/src/macros/functions/auth_registry.nr

```diff
@@ -1,3 +1,6 @@
+// TODO(benesjan): Instead of having this registry, just generate the authorize check when the given
+// `__aztec_nr_internals__` function is being generated. Having this registry is completely unnecessary.
```

Comment:
This registry is also unnecessary. Plan on tackling this once the stack is merged. (will go through all the "TODO(benesjan)" in the codebase once that's done and do the cleanups).